doc/source/multiscene.rst

```diff
@@ -40,6 +40,46 @@ iteratively overlays the remaining datasets on top.
     >>> blended_scene = new_mscn.blend()
     >>> blended_scene.save_datasets()
 
+Grouping Datasets
```

Comment:
I'm worried about this being confused with `group_files` which is used from `MultiScene.from_files` and is mentioned in the last section of this document. Would "Grouping Similar Datasets" be too awkward? Or should there maybe be a `from_files` specific section?

Reply:
No, "Grouping Similar Datasets" is much better. We could add two examples of how to create a multi scene (constructor, from_files) to the beginning of the chapter?

Reply:
Oh that's a good idea. Creation examples.

Reply:
Added a chapter on multi scene creation